packages/imba-monarch/src/utils.imba

```diff
@@ -193,10 +193,9 @@ export const ToImbaMap = {
 	'Ψ': '#'
 	'Γ': ''
 	'α': '@'
-	'ς': ''
 }
 
-const toImbaRegex = new RegExp("[ΞΦΨΓας]","gu")
+const toImbaRegex = new RegExp("[ΞΦΨΓα]","gu")
```

Comment:
Seems like there was partial inclusion of changes in a recent push using a different Greek letter. I believe these should be removed